python_modules/dagster/dagster/_generate/templates/PROJECT_NAME_PLACEHOLDER/cloud_workspace.yaml.tmpl

```diff
@@ -0,0 +1,4 @@
+locations:
```

Comment:
Do we need the `cloud_workspace.yaml` for the default project? Or should it just be for `scaffold-deployment` project?

Reply:
was thinking that `dagster project scaffold` would be a wrapper of `dagster project scaffold-repository` (which creates the first code location / dagster repo) and `dagster project scaffold-deployment` (which scaffolds the deployment for cloud)


maybe - a better way around is to have dagster cloud cli that does the cloud deployment scaffold?

Reply:
ok, i decided to take the cloud yaml out of the default project template. and i opened a thread to discuss the deployment cli.